proposal/proposal.md

```diff
@@ -242,6 +246,8 @@ On a touch screen, panning and zooming are achieved with 'pinch' gestures.
 
 An alternative will be needed for mice that don't have scroll wheels, eg keyboard shortcuts and right-click menu options. Another possibility would be to make a double-click 'zoom-in' by 2x, and a shift-double-click 'zoom-out', or buttons for zoom in/out/reset.
 
+Horizontal and vertical scroll bars can also be added, it will allow to keep the track of how much zoomed the image is and which region of the image is currently displayed under the image area with respect to the whole image. In preference settings, user can show/hide the scroll bars as per the need and personal preference.

```

Comment:
If you don't mind, could you move this to the optional extras section at the end of the document. It's something we could look to implement once they've put the basic preferences scaffolding in place (unless they have enough time int he PoC to do it of course).

Reply:
Sure, it's done.